lib/internal/Magento/Framework/Model/EntitySnapshot.php

```diff
@@ -48,14 +49,14 @@ public function __construct(
      * @param object $entity
      * @return void
      */
-    public function registerSnapshot($entityType, $entity)
+    public function registerSnapshot($entityType, $entity): void
```

Comment:
Unfortunately, we are not allowed to change public method signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)